compose-recipes/db2/service.db2.yaml

```diff
@@ -3,6 +3,7 @@ services:
         build: ./compose-recipes/db2
         container_name: ${NAMESPACE}-database-db2
         environment:
+            -   COMPOSER_DATABASE_NAME=lportal
             -   DB2INSTANCE=db2admin
             -   DB2INST1_PASSWORD=lportal
             -   LICENSE=accept
```

Comment:
Besides the DB name, is it possible that these other values will change? What is the use of having it them an env variable right now?

Reply:
Got it, thanks @anthony-chu 👍